Source/Core/Core/ConfigLoaders/BaseConfigLoader.cpp

```diff
@@ -75,6 +75,7 @@ const std::map<Config::System, int> system_to_ini = {
     {Config::System::GFX, F_GFXCONFIG_IDX},
     {Config::System::Logger, F_LOGGERCONFIG_IDX},
     {Config::System::Debugger, F_DEBUGGERCONFIG_IDX},
+    {Config::System::SYSCONF, F_DOLPHINCONFIG_IDX},
```

Comment:
So we're going to take the existing `Config::System::SYSCONF` and have it load both from the SYSCONF file and from Dolphin.ini which already is used by `Config::System::Main`? Don't get me wrong, SYSCONF settings would be very nice to have in the Android GUI, but this way of doing it seems messy... In particular, you'd end up having a situation where one of the files overrides the other (I assume Dolphin.ini overrides SYSCONF), which is probably going to be confusing.

Reply:
Dolphin always overrides SYSCONF on loading wii game. 
Config::System::SYSCONF doesn't save to setting file before, and this pr save Config::System::SYSCONF settings to Dolphin.ini.

Reply:
> Config::System::SYSCONF doesn't save to setting file before

While it didn't save to an INI file before, it did save to the SYSCONF file. I think the proper solution would be if the Android GUI would write to the SYSCONF file, and it would be especially good if it could do so by going through the new config system rather than having to write Java code for handling the SYSCONF file. But this is harder to do, which is probably why it hasn't been implemented in Dolphin yet...

> with this changes, we have per game sysconf settings.

It was already possible to set SYSCONF settings in game INIs. `GetINIToSectionMap` only exists for section name backwards compatibility – it is possible to use game INI settings that aren't included there.